status-im/status-react
# Add support for SOCKS token

fixes #11979 

### Summary

Adding Unisocks to our token list

![](https://user-images.githubusercontent.com/18485527/114413641-8c7e2e00-9b84-11eb-8a22-536e9ff6c4a5.png)

![](https://user-images.githubusercontent.com/18485527/114413653-8ee08800-9b84-11eb-8e91-2a835aad0602.png)

#### Platforms

- Android
- iOS

##### Functional

- wallet / transactions

### Steps to test

- Open Status
- Go to wallet
- Open option
- Select Manage assets
- Verify that Unisocks (SOCKS) is available to select and use

status: ready
